meshtastic/tests/test_node_runtime_channel_request.py

```
@@ -0,0 +1,281 @@
+"""Unit tests for channel_request_runtime module."""
+
+import logging
+import threading
+from unittest.mock import MagicMock
+
+import pytest
+from pytest import LogCaptureFixture
```

Ruff (0.15.6)

[warning] Incorrect import of `pytest`; use `import pytest` instead

(PT013)